src/Rings/mpoly-ideals.jl

```diff
@@ -907,11 +907,11 @@ ideal(x^2)
 julia> J = ideal(R, [x, y])^2
 ideal(x^2, x*y, y^2)
 
-julia> issubset(I, J)
+julia> is_subset(I, J)
 true
 ```
 """
-function Base.issubset(I::MPolyIdeal{T}, J::MPolyIdeal{T}) where T
+function is_subset(I::MPolyIdeal{T}, J::MPolyIdeal{T}) where T
```

Comment:
Question: Does this behave well, if a user mistakenly writes issubset(I,J) instead of is_subset(I,J)? 
                   (i.e. does it correctly give an error message or does it simply do something unexpected)

Reply:
They are aliases of each other and behave 100% identical